src/expressServer.js

```diff
@@ -133,6 +134,7 @@ app.use('/api/v1/orders', orderRouter /* #swagger.tags = ['Orders'] */);
 app.use('/api/v1/reviews', reviewRouter /* #swagger.tags = ['Reviews'] */);
 app.use('/api/v1/wishlist', wishlistRoutes /* #swagger.tags = ['Wishlist'] */);
 app.use('/api/v1/chat', chatRoutes);
+app.use('api/v1/recommendations', recommendationRoutes);
```

Comment:
```suggestion
app.use('/api/v1/recommendations', recommendationRoutes);
```

without the leading "slash" this is currently defining a route at: `localhost:8000api/v1/recommendations/...` instead of `localhost:8000/api/v1/recommendations/...`